src/github/client/app-token-holder.test.ts

```diff
@@ -17,17 +17,17 @@ describe("AppTokenHolder", () => {
 		const appTokenHolder = new AppTokenHolder();
```

Comment:
Maybe some test on the different path of the new key locator?

Reply:
@gxueatlassian,  moved key locator into seperate file and written test as well.